spring-cloud-stream-tuple/src/main/java/org/springframework/cloud/stream/tuple/kryo/TupleKryoRegistrar.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.tuple.kryo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.integration.codec.kryo.AbstractKryoRegistrar;
```

Comment:
since this depends on spring-integration, should it be in the `integration` subpackage - or perhaps even `integration.kryo` - the main question would be if this is itself a candidate for moving to spring-integration along with the other classes in the `integration` package